### ISSUE TITLE
REQUEST: Multiple Selection

### ISSUE THREAD

**Issue by eventhandler23** [nisrulz/recyclerviewhelper#7]
An easy way to multiple select an item on the recyclerview 👍

**Comment by SubhrajyotiSen** [comment]
Can I take up this issue?

**Comment by nisrulz** [comment]
@SubhrajyotiSen : yeah go ahead. Please send your PR to only `develop` branch and make sure you are in synch with it before you begin.
